server/sonar-web/src/main/js/components/workspace/WorkspacePortal.tsx

```diff
@@ -17,31 +17,29 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-import Backbone from 'backbone';
+import * as React from 'react';
+import { createPortal } from 'react-dom';
 
-export default Backbone.Model.extend({
-  validate() {
-    if (!this.has('__type__')) {
-      return 'type is missing';
-    }
-    if (this.get('__type__') === 'component' && !this.has('key')) {
-      return 'key is missing';
-    }
-    if (this.get('__type__') === 'rule' && !this.has('key')) {
-      return 'key is missing';
-    }
-  },
+interface Props {}
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: An empty interface is equivalent to `{}`. ([typescript:S4023](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS4023))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3140&issues=AWIKRnnMGvJAiVQc9aQn&open=AWIKRnnMGvJAiVQc9aQn)

[ISSUE_KEY]: AWIKRnnMGvJAiVQc9aQn